backend/gameplay/consumers.py

```
@@ -436,48 +424,34 @@ async def connect(self):
-			self.get_game_attribute(
-				chess_game_model, "parsed_board_placement"),
-			self.get_game_attribute(
-				chess_game_model, "en_passant_target_square")
+			self.get_game_attribute(chess_game_model, "parsed_board_placement"),
+			self.get_game_attribute(chess_game_model, "en_passant_target_square")
 		)
 
-		parsed_move_data: dict = json.loads(event["move_data"])
+		parsed_move_data = json.loads(text_data)
+		piece_color = parsed_move_data["piece_color"]
+		opposing_color = get_opposite_color(piece_color.lower())
```

Ruff (0.8.2)

`get_opposite_color` may be undefined, or defined from star imports

(F405)